test-e2e/panels/filters.test.ts

```diff
@@ -1,165 +1,183 @@
-import { test, expect } from '@playwright/test';
```

Comment:
Could you checked what happened with this commit ?

The end of line is expected to by LF (ascii 10) and in this commit it is now CR + LF (ascii 13 + ascii 10)

Reply:
fixed